sympy/polys/polyoptions.py

```diff
@@ -722,6 +722,16 @@ def preprocess(cls, method):
             raise OptionError("expected a string, got %s" % method)
 
 
+class Fraction(with_metaclass(OptionType, Flag)):
```

Comment:
The option `fraction` is not for polynomials. It should not be necessary to define it here. It can be handled directly in `factor` and other private factoring methods.